BandNameGenerator/ViewModel.cs

```diff
@@ -0,0 +1,54 @@
+﻿using BandNameGenerator.Genres;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BandNameGenerator
+{
+    public class ViewModel
+    {
+        public enum Genre
+        {
+            Metal,
+            Country,
+            Folk
+        }
+
+        public ViewModel()
+        {
+        }
```

Comment:
Can remove empty constructor?

Reply:
Yes! At one point that was doing something, but not anymore. :)